pt/cheatsheet-unsupervised-learning.md

```diff
@@ -1,340 +1,339 @@
 
 <br>
 
 **26. Silhouette coefficient ― By noting a and b the mean distance between a sample and all other points in the same class, and between a sample and all other points in the next nearest cluster, the silhouette coefficient s for a single sample is defined as follows:**
 
-&#10230;
+&#10230; Coeficiente de silhueta ― Ao indicar a e b a distância média entre uma amostra e todos os outros pontos na mesma classe, e entre uma amostra e todos os outros pontos no grupo mais próximo, o coeficiente de silhueta s para uma única amostra é definida como se segue:
 
 <br>
 
 **27. Calinski-Harabaz index ― By noting k the number of clusters, Bk and Wk the between and within-clustering dispersion matrices respectively defined as**
 
-&#10230;
+&#10230; Índice Calinski-Harabaz ― Indicando por k o número de grupos, Bk e Wk as matrizes de disperção entre e dentro do agrupamento respectivamente definidos como
 
 <br>
 
 **28. the Calinski-Harabaz index s(k) indicates how well a clustering model defines its clusters, such that the higher the score, the more dense and well separated the clusters are. It is defined as follows:**
 
-&#10230;
+&#10230; o índice Calinski-Harabaz s(k) indica quão bem um modelo de agrupamento define o seu grupo, tal que maior a pontuação, mais denso e bem separado os grupos estão. Ele é definido como a seguir:
 
 <br>
 
 **29. Dimension reduction**
 
-&#10230;
+&#10230; Redução de dimensão
 
 <br>
 
 **30. Principal component analysis**
 
-&#10230;
+&#10230; Análise de componente principal
 
 <br>
 
 **31. It is a dimension reduction technique that finds the variance maximizing directions onto which to project the data.**
 
-&#10230;
+&#10230; É uma técnica de  redução de dimensão que encontra direções de maximização de variância em que projetam os dados.
 
 <br>
 
 **32. Eigenvalue, eigenvector ― Given a matrix A∈Rn×n, λ is said to be an eigenvalue of A if there exists a vector z∈Rn∖{0}, called eigenvector, such that we have:**
 
-&#10230;
+&#10230; Autovalor, autovetor ― Dada uma matriz A∈Rn×n, λ é dito ser um autovalor de A se existe um vetor z∈Rn∖{0}, chamado autovetor, tal que temos:
 
 <br>
 
 **33. Spectral theorem ― Let A∈Rn×n. If A is symmetric, then A is diagonalizable by a real orthogonal matrix U∈Rn×n. By noting Λ=diag(λ1,...,λn), we have:**
 
-&#10230;
+&#10230; Teorema espectral - Seja A∈Rn×n. Se A é simétrica, então A é diagonizável por uma matriz ortogonal U∈Rn×n. Denotando Λ=diag(λ1,...,λn), temos:
 
 <br>
 
 **34. diagonal**
 
-&#10230;
+&#10230; diagonal
 
 <br>
 
 **35. Remark: the eigenvector associated with the largest eigenvalue is called principal eigenvector of matrix A.**
 
-&#10230;
+&#10230; Observação: o autovetor associado com o maior autovalor é chamado de autorvetor principal da matriz A.
```

Comment:
- Observação: o autovetor associado com o maior autovalor é chamado de **autovetores** principal da matriz A.

Reply:
OPS! Correct is **autovetor**